.github/workflows/ci.yml

```diff
@@ -14,19 +14,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Set up JDK 11
+      - name: Set up JDK 17
         uses: actions/setup-java@v1
         with:
-          java-version: 11
-      - name: Cache Dependencies
-        uses: actions/cache@v2
+          java-version: 17
+      - name: Build with Gradle
+        uses: gradle/gradle-build-action@v1
         with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-          restore-keys: |
-            ${{ runner.os }}-maven
-      - name: Build with Maven
-        run: mvn -B package --file backend/pom.xml
+          arguments: assemble
```

Comment:
Why do you use `assemble` instead of `build`?

Reply:
`build` also executes tests, but because there are no tests, this isn't necessary. So we save a second or so